foundation/org.eclipse.persistence.core/src/org/eclipse/persistence/config/EntityManagerProperties.java

```diff
@@ -319,6 +319,21 @@ public class EntityManagerProperties {
      * )
      */
     public static final String COMPOSITE_UNIT_PROPERTIES = PersistenceUnitProperties.COMPOSITE_UNIT_PROPERTIES;
+    
```

Comment:
I intended them to be Entity Manager Properties, since we could allow them to be changed in a transaction. I believe the PersistenceUnit already has support for this feature, but it can't be changed dynamically, hence the reason for this enhancement. I hope that makes sense! Please let me know if it doesn't or if my reasoning is not correct.